docs/docs/clients/server-side.md

```diff
@@ -515,21 +515,21 @@ buttonData, _ := flags.GetFeatureValue("secret_button")
 <TabItem value="rust" label="Rust">
 
 ```rust
-use flagsmith_flag_engine::identities::Trait;
+use flagsmith::models::SDKTrait;
 use flagsmith_flag_engine::types::{FlagsmithValue, FlagsmithValueType};
 
 let identifier = "delboy@trotterstraders.co.uk";
 
-let traits = vec![Trait {
-            trait_key: "car_type".to_string(),
-            trait_value: FlagsmithValue {
+let traits = vec![SDKTrait::new(
+            "car_type".to_string(),
+            FlagsmithValue {
                 value: "robin_reliant".to_string(),
                 value_type: FlagsmithValueType::String,
             },
-        }];
+        )];
 
 // The method below triggers a network request
-let identity_flags = flagsmith.get_identity_flags(identifier, Some(traits)).unwrap();
+let identity_flags = flagsmith.get_identity_flags(identifier, Some(traits), None).unwrap();
```

Comment:
`None` seems odd here? Shouldn't this be a Boolean?

Reply:
Argument type is `Option<bool>`, so it should be one of:
- `None`
- `Some(true)`
- `Some(false)`

`Option` type is used to convey that the argument is non-essential, and the user can avoid making an explicit decision by passing `None`.

Reply:
Fair enough, it just seems a bit odd to explicitly set `None` instead of `false`. It's not important though, so I'll just approve.